awx/ui_next/src/screens/Setting/shared/data.allSettingOptions.json

```diff
@@ -2745,29 +2993,6 @@
               "category_slug": "system",
               "default": true
           },
-          "PENDO_TRACKING_STATE": {
-            "default": "off",
-            "type": "choice",
-            "required": true,
-            "label": "User Analytics Tracking State",
-            "help_text": "Enable or Disable User Analytics Tracking.",
-            "category": "UI",
-            "category_slug": "ui",
-            "choices": [
-                [
-                    "off",
-                    "Off"
-                ],
-                [
-                    "anonymous",
-                    "Anonymous"
-                ],
-                [
-                    "detailed",
-                    "Detailed"
-                ]
-            ]
-        },
```

Comment:
@egmar it looks like this whole deleted code block for `PENDO_TRACKING_STATE` needs to go back into the file; the failing UI unit test should find it and CI should hopefully pass after reverting this deletion.

Reply:
Looks ok after adding `PENDO_TRACKING_STATE`.